media-autobuild_suite.bat

```diff
@@ -1579,6 +1579,16 @@ for /f "tokens=2" %%P in ('tasklist /v ^|findstr MABSbat') do set ourPID=%%P
 if exist %build%\compilation_failed del %build%\compilation_failed
 if exist %build%\fail_comp del %build%\compilation_failed
 
+REM Test mklink availability
+set symlinkSupport=""
+mkdir testmklink
+mklink /d linkedtestmklink testmklink
+if %ERRORLEVEL%==0 (
+    set symlinkSupported="winsymlinks:nativestrict"
```

Comment:
Does msys have a default setting other than ""?

Reply:
Usually not, unless you override it yourself in the .ini files inside /msys64.